apps/openmw/mwworld/weather.cpp

```diff
@@ -289,44 +294,54 @@ WeatherManager::WeatherManager(MWRender::RenderingManager* rendering, MWWorld::F
 
     //Weather
     Weather clear;
+    clear.mObstructsCelestialBodies = false;
```

Comment:
After trying more values, I think it actually may describe how quickly to transition the clouds to the new texture. If you set small values, you'll see that the clouds transition quickly, but not immediately, to the new texture. It doesn't appear to have an impact when transitioning from a 0.0 -> 1.0, so I think it only matters what the target weather pattern is, not what it was.

Rather than continue to clutter the request with more research findings, I'd prefer to take this to the forums and eventually submit a pull request later, once it is understood a little better.